models/cachetime.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+import "time"
+
+type CacheTime struct {
+	ID           string    `json:"_id" bson:"_id"`                   // MD5 of the path
+	Path         string    `json:"path" bson:"path"`                 // Path for which caching is set
+	ETag         string    `json:"etag" bson:"etag"`                 // ETag for cache validation
+	CollectionID int       `json:"collectionId" bson:"collectionId"` // Collection ID
+	ReleaseTime  time.Time `json:"releaseTime" bson:"releaseTime"`   // Release time in ISO-8601 format
```

Comment:
We prefer `snake_case` for JSON - https://github.com/ONSdigital/dp-standards/blob/main/API_STANDARDS.md?plain=1#L33